src/app/app.component.ts

```diff
@@ -66,7 +66,8 @@ import {ItemComponent} from './dspace/components/item.component';
         { path: "/communities/:id/:page", name: "Communities", component: CommunityComponent },
         { path: "/collections/:id", name: "Collections", component: CollectionComponent },
         { path: "/collections/:id/:page", name: "Collections", component: CollectionComponent },
-        { path: "/items/:id", name: "Items", component: ItemComponent }
+        { path: "/items/:id", name: "Items", component: ItemComponent },
+        { path: "/items/:id/:page", name: "Items", component: ItemComponent }
```

Comment:
The page variable could be leveraged for displaying the full view. Thanks for reviewing the code. Any constructive criticism is welcome.